qemu/tests/fio_perf.py

```diff
@@ -209,7 +209,7 @@ def fio_install(tarball):
                 if s:
                     raise exceptions.TestFail("Failed to online disk: %s" % o)
     # format disk
-    if format:
+    if format == "True":
```

Comment:
>>> params = {'format': 'True', 'vga': 'cirrus', 'cpu_model_flags': ''}
>>> format = params.get("format")
>>> type(format)
<type 'str'>

So it is a str indeed, and you can have a try with previous "if format", it will fail.
Besides, here are logs when format=false, we can see the mkfs.xfs is excuated although format=false. What I want is that mkfs.xfs should not be excuated when format=false.
2017-07-19 04:05:56,936 output           L0655 DEBUG| yama_format: False
2017-07-19 04:05:56,936 client           L1078 DEBUG| Sending command: i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt